usage-rules.md

```diff
@@ -1128,3 +1150,17 @@ When testing resources:
 - Use `authorize?: false` in tests where authorization is not the focus
 - Write generators using `Ash.Generator`
 - Prefer to use raising versions of functions whenever possible, as opposed to pattern matching
+
+## Debugging
+
+Use `dbg` instead of `IO.puts` or `Logger.*` for debugging purposes
+
+## Ash.CiString
```

Comment:
This is too much real estate just for CiString.

We can mention them, but needs to be *way* less, like one or two lines max.

We can say something like:

> attributes of type `:ci_string` will have their values represented with `%Ash.CiString{}`. These cannot be directly compared, use `Comp.equal?(ci_string, string_or_ci_string)`. See the docs in `Ash.CiString` for more.

We can say things like "see the docs of X", but can't assume how the agent will do that.